src/components/aboutUs/components/TeamProfile.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Avatar, Card, Icon, List } from 'antd';
+
+import { ABOUT_US, TEAM_INFO } from './constants';
+
+const { Meta } = Card;
+const { 
+    CLASSNAMES: { AVATAR, TEAM_CARD, TEAM_PROFILE },
+    STRINGS: { FACEBOOK, GITHUB, ICON, LINKEDIN, SQUARE }, 
+} = ABOUT_US;
+
+class TeamProfile extends React.Component {
+    render() {
+        return(
+            <List  
+                bordered={false}
+                dataSource={TEAM_INFO}
+                grid={{ gutter: 12, lg: 4, md: 4, sm: 1.8, xs: 1.8, xxl: 3 }}
+                renderItem={item => (
+                    <List.Item>
+                        <div key={item.name}>
+                            <Card
+                                hoverable
+                                bordered={false}
+                                className={TEAM_CARD}
+                                cover={
+                                    <Avatar src={item.photo} shape={SQUARE} className={AVATAR} />
+                                }
+                                actions={[<Icon type={LINKEDIN} key={ICON} />, 
+                                    <Icon type={FACEBOOK} key={ICON} />, 
+                                    <Icon type={GITHUB} key={ICON} />]}
+                            >
+                                <Meta
+                                    title={item.name}
+                                    description={<p className={TEAM_PROFILE} >
+                                        {item.description}
+                                    </p>}
```

Comment:
clean up code
```
description={
    <p className={TEAM_PROFILE} >
        {item.description}
    </p>
}
```

Reply:
actually remove the `p` tag around `item.description`. Allow the card element to style the description content

Reply:
ok